samples/AspNetIntegration/AspNetIntegration.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.3-preview2" />
```

Comment:
Do we usually use the preview version packages in the samples? I thought we didn't.

Reply:
No, but the bits needed are unfortunately only in the preview package.